docs/iot-gateway/getting-started.md

```diff
@@ -1,36 +1,75 @@
 ---
 layout: docwithnav-gw
 title: Getting started with ThingsBoard IoT Gateway
-description: Write your first IoT project using ThingsBoard IoT Gateway
+description: Configure MQTT, OPC-UA, and Modbus connectors to establish connections with their respective demo servers in the Docker container and retrieve data.
 
 ---
 
 * TOC
 {:toc}
 
-This guide covers initial IoT Gateway installation and configuration.
-We will connect IoT Gateway to ThingsBoard server, control it and visualize some basic gateway statistics: the amount of devices connected and messages processed.
-We will also configure MQTT connector in order to subscribe to device data feed from external devices.  
+The ThingsBoard IoT Gateway is an open-source solution, designed to serve as a bridge between IoT devices connected to 
+legacy and third-party systems with ThingsBoard.
 
+This guide covers initial IoT Gateway installation and configuration, we will do the following things:
+- Create a new gateway device using [ThingsBoard IoT Gateways dashboard](#prerequisites);
+- Launch the gateway using Docker command;
+- Configure different connector types ([MQTT](/docs/iot-gateway/config/mqtt/), [OPC-UA](/docs/iot-gateway/config/opc-ua/), [Modbus](/docs/iot-gateway/config/modbus/)) in order to connect to a local demo servers and read data from them;
+- Check received device data on ThingsBoard.
 
-### Prerequisites
+## Prerequisites
 
-If you don't have access to a running ThingsBoard instance, use either [**Live Demo**](https://thingsboard.cloud/signup) or
-[**Installation Guide**](/docs/user-guide/install/installation-options/) 
-to fix this. 
+- Before initiating the Gateway setup, ensure that the ThingsBoard server is up and running. The simplest approach is to utilize the [Live Demo](https://demo.thingsboard.io) or [ThingsBoard Cloud](https://thingsboard.cloud). Alternatively, you can install ThingsBoard manually by following the steps outlined in the [Installation Guide](/docs/user-guide/install/installation-options/).
+- Before moving forward, ensure Docker is installed and properly configured on your machine. If you haven't installed Docker yet, you can download it from the [official Docker website](https://docs.docker.com/engine/install/) and follow their installation guide for your specific operating system. 
+- If you don't have a dashboard installed, you can download Gateway widget bundle [JSON file here](/docs/iot-gateway/resources/thingsboard-gateway-widget-bundle.json){:target="_blank" download="thingsboard-gateway-widget-bundle.json"} and ThingsBoard IoT Gateways dashboard [JSON file here](/docs/iot-gateway/resources/thingsboard-gateways-dashboard.json){:target="_blank" download="thingsboard-gateways-dashboard.json"}. You can do this using the section below:
```

Comment:
Please mention the minimum required version of ThingsBoard. 

Also, I would add something like "Starting ThingsBoard 3.6.x the dashboard is .. ".